scalanet/src/io/iohk/scalanet/codec/SeqCodec.scala

```diff
@@ -0,0 +1,43 @@
+package io.iohk.scalanet.codec
+
+import scodec.bits.BitVector
+import scodec.{Attempt, Codec, DecodeResult, SizeBound}
+
+import scala.annotation.tailrec
+
+class SeqCodec[A](implicit subcodec:Codec[A], intCodec:Codec[Int]) extends Codec[Seq[A]] {
```

Comment:
You can derive seq codec form scodec list codec like:
```
    implicit def seqCoded[A](implicit listCodec: Lazy[Codec[List[A]]]): Codec[Seq[A]] = {
      listCodec.value.xmap(l => l.toSeq, seq => seq.toList)
    }
```

it is done this way in `io.iohk.scalanet.codec.DefaultCodecs.General` there are also codec for different types of adresses